linked-list.rb

```diff
@@ -63,9 +63,26 @@ def display
    end
 
    def include?(key)
+      current = @head
+      full_list = []
+      return p "#{key} is included!" if current.value == key
+      while current.next_node != nil
+          current = current.next_node
+          return p "#{key} is included!" if current.value == key
+      end
+      return p "#{key} is not included."
+
    end
 
    def size
+      current = @head
+      full_list = []
+      while current.next_node != nil
+          full_list += [current.value.to_s]
+          current = current.next_node
+      end
+      full_list += [current.value.to_s]
+      return p "Size is #{full_list.length}."
    end
```

Comment:
These looks generally good -- however, both methods are creating a side effect of printing something. If you are not asked to print something out, then you don't necessarily want to because it limits how you can use that method in the future.

I would expect that a method with a `?` at the end would return a boolean and not print anything out. 

I would also expect that a size method would only return a single numerical value and nothing else.